bin/setup

```diff
@@ -20,11 +20,6 @@ chdir APP_ROOT do
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
 
-  puts "\n== Copying sample files =="
```

Comment:
I'm in favour of keeping this step. The purpose of the `bin/setup` and `bin/update` scripts are to automate the steps needed to bootstrap the application.

Reply:
OK. New version hopefully avoids file clutter.